hw1/seq_transformer.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Sep 27 18:39:56 2022
+
+@author: anja1
+
+Программа, которая в бесконечном цикле считывает команды от
+пользователя. После команды программа запрашивает у пользователя
+последовательность нуклеиновой кислоты и распечатывает результат.
+
+Список команд:
+    exit — завершение исполнения программы
+    transcribe — напечатать транскрибированную последовательность
+    reverse — напечатать перевёрнутую последовательность
+    complement — напечатать комплементарную последовательность
+    reverse complement — напечатать обратную комплементарную последовательность
+
+
+
+"""
+
+# Словари для транскибции и комплиментации
+dic_tr = {'A':'U','T':'A','G':'C', 'C':'G',
+          'a':'u','t':'a','g':'c','c':'g'}
+
+dic_com_DNA = {'A':'T','T':'A','G':'C', 'C':'G',
+               'a':'t','t':'a','g':'c','c':'g'}
+
+dic_com_RNA = {'A':'U','U':'A','G':'C', 'C':'G',
+               'a':'u','u':'a','g':'c','c':'g'}
+
+# Список разрешенных символов на вхождение
+allow = ['a','u','t','g','c','A','T','G','C','U']
```

Comment:
```suggestion
allow = {'a','u','t','g','c','A','T','G','C','U'}
```
Для проверки на вхождение удобнее всего использовать множества. Поиск в них осуществляется в среднем за константное время O(1). В списках же за O(n)